OpenRA.Mods.Common/Traits/Attack/AttackGarrisoned.cs

```diff
@@ -161,7 +161,7 @@ public override void DoAttack(Actor self, Target target)
 				if (port == null)
 					return;
 
-				var muzzleFacing = targetYaw.Angle / 4;
+				var muzzleFacing = targetYaw.Facing;
```

Comment:
I guess this removes the hard-coded case of a building with 4 sides?

Reply:
Facing is defined as 1/4 the angle - nothing changes with the value, but this uses the correct property so that it is clear that this is using a facing not an angle.